File: app/src/main/java/com/shopcart/Banner.java
package com.shopcart;

public class Banner {
    private String header;
    private String body;
    private String photoUrl;

    public Banner() {
    }

    public Banner(String header, String body, String photoUrl) {
        this.header = header;
        this.body = body;
        this.photoUrl = photoUrl;
    }

    public String getHeader() {
        return header;
    }

    public void setHeader(String header) {
        this.header = header;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getPhotoUrl() {
        return photoUrl;
    }

    public void setPhotoUrl(String photoUrl) {
        this.photoUrl = photoUrl;
    }
}

File: app/src/main/java/com/shopcart/data/models/Banner.kt
package com.shopcart.data.models

class Banner {
    var header: String? = null
    var body: String? = null
    var photoUrl: String? = null

    constructor()
    constructor(header: String?, body: String?, photoUrl: String?) {
        this.header = header
        this.body = body
        this.photoUrl = photoUrl
    }
}